docs/guides/python/integration_tests.md

```diff
@@ -0,0 +1,64 @@
+## :fontawesome-solid-flask-vial: Running integration tests
+
+For Python package-based projects it's pretty easy to setup the integration tests pipeline with the instruction below.
+
+!!! hint
+
+    For Notebook-based workflows, take a look at the [nutter project](https://github.com/microsoft/nutter).
+
+
+## :octicons-file-code-24:  Preparing the configuration
+
+* Add an entrypoint file to the `tests/entrypoint.py`
+
+```python title="tests/netrypoint.py"
+import sys
+
+import pytest
+
+if __name__ == '__main__':
+    pytest.main(sys.argv[1:])
+```
+
+* Add your tests to the `tests/integration` folder
+* Configure the special test-oriented `workflow`:
+
+```yaml title="conf/deployment.yml"
```

Comment:
Don't you need other parts you described in the SO answer?

```
custom:
  basic-cluster-props: &basic-cluster-props
    spark_version: "10.4.x-cpu-ml-scala2.12"

  basic-static-cluster: &basic-static-cluster
    new_cluster:
      <<: *basic-cluster-props
      num_workers: 1
      node_type_id: "{{some-node-type}}"

build:
  commands:
    - "poetry build -f wheel" #safe to use inside poetry venv
```

Reply:
This one has been fixed in #446 and released in 0.7.4